pages/doc/integrations_aws_overview.md

```diff
@@ -225,22 +227,26 @@ To set up an AWS integration, you must have a [Role ARN](#give-read-only-access-
      
 1. Click **Register**. 
 
-The integration is added to the Amazon Web Services Integrations list. If you want to configure allow lists and refresh rate for the CloudWatch integration, click the **CloudWatch** link in the Types column and follow the instructions in [Configuring CloudWatch Data Ingestion](integrations_aws_metrics.html#configuring-cloudwatch-data-ingestion).
+The AWS integration, which includes a CloudWatch, an AWS Metrics+, and, optionally, a CloudTrail integration, are added to the Amazon Web Services integrations list on the **Setup** tab.
+
+- If you want to configure allow lists and refresh rate for the CloudWatch ingestion, click the **CloudWatch** link in the Types column and follow the instructions in [Configuring CloudWatch Data Ingestion](integrations_aws_metrics.html#configuring-cloudwatch-data-ingestion).
+- If you want to disable the ingestion of support service limit metrics (also known as service quotas), that is, the metrics with the namespace `aws.limits.*`, click the **AWS Metrics+** link in the Types column and follow the instructions in [Disable Service Limit Metrics Ingestion](integrations_aws_metrics.html#disable-service-limit-metrics-ingestion).
 
 
-### Register Additional Amazon Web Services
+### Register Additional AWS Services to an AWS Integration
 
-After you set up the AWS integration with a [Role ARN](#give-read-only-access-to-your-amazon-account-and-get-the-role-arn), you can additionally register more Amazon Web services.
+After you set up the AWS integration with a [Role ARN](#give-read-only-access-to-your-amazon-account-and-get-the-role-arn), you can additionally register more CloudWatch, AWS Metrics+, or CloudTrail integrations associated with the same role ARN.
 
 1. In your service instance (`https://<your_instance>.wavefront.com`), click **Integrations** on the toolbar.
 1. In the Featured section, click the **Amazon Web Services** tile.
 1. Click the **Setup** tab.
 1. Click the **Advanced** link.
-1. In the Cloud Integration page, click **Add Amazon Web Services** and select an option:
+1. On the Cloud Integrations page, click **Add Cloud Integration &gt; Add Amazon Web Services** and select an option:
    * To register an AWS Metrics+ service, select **Register AWS Metrics+**, and configure the following integration properties:
       
       1. **Name** -- Name to identify the integration.
       2. **Role ARN** -- Select the Role ARN from your Amazon account.
+      3. **Service Limit Metrics** -- Disable or enable the ingestion of support service limit metrics (also known as service quotas), that is, the metrics with the namespace `aws.limits.*`. You can later edit this setting, if necessary.
```

Comment:
**Service Limit Metrics** -- Disable or enable the ingestion of support service limit metrics (also known as service quotas), that is, the metrics with the namespace `aws.limits.*`. You can later edit this setting if necessary.